### PR TITLE
Fix `Frankfurter Rundschau`

### PR DESCRIPTION
**Pull request by addie9800**
This PR adds parsing for this article: https://www.fr.de/ratgeber/tiere/zeitumstellung-tiere-haustiere-hunde-katzen-wildtier-landwirtschaft-tagesrhythmus-zr-93379931.html

### CODE REVIEW

File: src/fundus/publishers/de/frankfurter_rundschau.py [hunk 1]
```diff
@@ -13,9 +14,15 @@
 
 class FrankfurterRundschauParser(ParserProxy):
     class V1(BaseParser):
-        _paragraph_selector = CSSSelector("p.id-StoryElement-paragraph, article > ul > li")
-        _summary_selector = CSSSelector("p.id-StoryElement-leadText")
-        _subheadline_selector = CSSSelector("h2.id-StoryElement-crosshead")
+        _paragraph_selector = XPath(
+            "//p[@class='id-StoryElement-paragraph'] | "
+            "//p[contains(@class,'id-Article-content-item-paragraph') and text()] |"
+            "//div[@class='id-Article-body']//ul/li[not(@class='id-AuthorList-item ')]"
```
Comment (MaxDall) [inline src/fundus/publishers/de/frankfurter_rundschau.py:20]: This one differs from the original `article > ul > li` is there a reason for that?

File: src/fundus/publishers/de/frankfurter_rundschau.py [hunk 2]
```diff
@@ -13,9 +14,15 @@
...
+        )
+        _summary_selector = CSSSelector(
+            "p.id-StoryElement-leadText, " "p[class='id-Article-content-item id-Article-content-item-summary']"
```
Comment (MaxDall) [inline src/fundus/publishers/de/frankfurter_rundschau.py:23]: There are unnecessary quotation marks

**Review by MaxDall — CHANGES_REQUESTED** [review]

File: src/fundus/publishers/de/frankfurter_rundschau.py (hunk 1 above)
Comment (addie9800) [reply src/fundus/publishers/de/frankfurter_rundschau.py:20]: Yes, they added another list item in some articles called "Meine News". You can see an example here: https://www.fr.de/politik/ampel-regierung-koalition-news-lindner-fdp-habeck-gruene-spd-wirtschaft-plan-papier-zr-93394586.html

File: src/fundus/publishers/de/frankfurter_rundschau.py (hunk 2 above)
Comment (addie9800) [reply src/fundus/publishers/de/frankfurter_rundschau.py]: Thanks, seems to be left-over from formatting.

**Review by MaxDall — APPROVED** [review]

**Pull request merged**